src/main/java/org/kohsuke/github/GHPullRequest.java

```diff
@@ -259,6 +284,34 @@ protected void wrapUp(GHPullRequestCommitDetail[] page) {
         };
     }
 
+    @Preview
+    @Deprecated
+    public GHPullRequestReview createReview(String body, @CheckForNull GHPullRequestReviewState event,
+                                            GHPullRequestReviewComment... comments)
+            throws IOException {
+        return createReview(body, event, Arrays.asList(comments));
+    }
+
+    @Preview
+    @Deprecated
+    public GHPullRequestReview createReview(String body, @CheckForNull GHPullRequestReviewState event,
+                                            List<GHPullRequestReviewComment> comments)
+            throws IOException {
+        if (event == null) {
+            event = GHPullRequestReviewState.PENDING;
+        }
+        List<DraftReviewComment> draftComments = new ArrayList<DraftReviewComment>(comments.size());
+        for (GHPullRequestReviewComment c : comments) {
+            draftComments.add(new DraftReviewComment(c.getBody(), c.getPath(), c.getPosition()));
+        }
+        return new Requester(root).method("POST")
+                .with("body", body)
+                //.with("event", event.name())
```

Comment:
Correct me if I'm wrong but if you aren't sending the event, then all of the reviews will be set to Pending state, right? If that's the case, why you have the event parameter in the first place?

I think we should let send it as it was specified in the method signature (2 cents opinion again)